INSTALL.md

```diff
@@ -1,11 +1,28 @@
-# Installing
+# Installation
 
-- [Installing from source](#installing-from-source)
-- [Installing from a package manager](#installing-from-a-package-manager)
+- [Install from a package manager](#install-from-a-package-manager)
+- [Install from source](#install-from-source)
```

Comment:
i've assumed that most people won't want to install from sources

Reply:
I'm unsure about this assumption. Installing from source is afaik still the only way that works regardless of what OS people are using. And Idris(2) is a research project, so I would assume the opposite: that most people install from the sources since they might want (or need) to modify them.

Reply:
> Installing from source is afaik still the only way that works regardless of what OS people are using

that's an interesting point

> And Idris(2) is a research project, so I would assume the opposite: that most people install from the sources since they might want (or need) to modify them.

interesting. I got my impression from asking on discord. I guess much of my reasoning for my changes has been to make it easier for people new to Idris to dive in, i.e. emphasising growing the community